pages/Documentation/Actions/bg-function(deprecated).md

```diff
@@ -0,0 +1,10 @@
+---
+title: bg-function(deprecated)
+---
```

Comment:
In general it's a good idea to have a new line separattion between front-matter and page content. Wonder if prettier can handle that?

Reply:
> new line separattion



> a new line separattion between front-matter and page content.

I didn't get the suggestion. Do you mean prettier to test if there is a separation `---` line?

Reply:
I mean a new line between second `---` separator and actual page content, but I can see that you have addressed this comment in https://github.com/browser-automation/cba-website/pull/18/commits/510fb12b9e3ee673230be539cb0fbb59625e8ddc So this LGTM.